.buildkite/pipeline.yml

```diff
@@ -3,7 +3,7 @@ steps:
     plugins:
       - JuliaCI/julia#v1:
           persist_depot_dirs: packages,artifacts,compiled
-          version: "1.7"
+          version: "1.12-nightly"
```

Comment:
Reminder to self that if this works we should change all `1.12-nightly`s to `1.12`, or even better a fixed version, just to avoid sudden changes under our feet.

Reply:
```suggestion
          version: "1.12"
```